docs/pages/enroll-resources/desktop-access/getting-started.mdx

```diff
@@ -17,6 +17,20 @@ Community Edition will not allow connections to any of them.
 
 Teleport Enterprise users can have unlimited number of desktops.
 
+## How it works
+
+The Teleport Desktop Service proxies traffic between the Teleport Proxy Service
+and remote Windows desktops in your infrastructure. Teleport users connect to a
+Windows desktop using Teleport Connect or the Teleport Web UI. These graphical
+clients use the proprietary Teleport Desktop Protocol to send traffic to and
+receive traffic from the Teleport Desktop Service (using WebSockets), which
```

Comment:
Nit: technically, Connect doesn't use WebSockets for that. I'm not sure if we need to mention the backing technology at all. ;)

Edit: Unless that's the point of this section to, uh, mention how exactly it works? Connect uses gRPC for that, but I'm just not sure if it's worth mentioning, same for WebSockets.

Reply:
Thanks! I was incorrectly applying my understanding of the Web UI. I'll remove the parentheses here.